api/tests/unit/features/multivariate/test_unit_multivariate_views.py

```
@@ -105,3 +106,32 @@ def test_list_mv_options__feature_in_other_project__returns_404(
+
+    # When
+    response = admin_client.patch(
+        url, data=json.dumps(data), content_type="application/json"
+    )
+
+    # Then
+    assert response.status_code == status.HTTP_200_OK
+    
+    mv_option.refresh_from_db()
+    assert mv_option.key == new_key
+    assert mv_option.default_percentage_allocation == initial_allocation
```

Ruff (0.16.1)

[warning] No newline at end of file

Add trailing newline

(W292)